shared/src/business/useCases/docketEntry/editPaperFilingInteractor.ts

```diff
@@ -56,6 +56,16 @@ export const editPaperFilingInteractor = async (
     docketEntryId: primaryDocumentFileId,
   });
 
+  let error;
+  if (!currentDocketEntry) {
+    error = new NotFoundError('Docket entry not found');
+  } else if (currentDocketEntry.servedAt) {
+    error = new Error('Docket entry has already been served');
+  }
+  if (error) {
```

Comment:
Can this be condensed to just throw inside the above if statements?

Reply:
Good catch! I made the change.